frontend/src/pages/analytics/event/AnalyticsEvent.tsx

```diff
@@ -406,28 +382,19 @@ const AnalyticsEvent: React.FC = () => {
         return;
       }
       setLoadingData(true);
-      setEmptyData(false);
       setLoadingChart(true);
       const { success, data }: ApiResponse<any> = await previewEvent(body);
+      setLoadingData(false);
+      setLoadingChart(false);
       if (success) {
```

Comment:
Add  condition of `data.visualIds.length>=2`

Reply:
add `data.visualIds.length === 2`